data-prepper-plugins/aws-sqs-common/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id 'java-library'
+    id "org.sonarqube" version "3.5.0.2730"
```

Comment:
This project does not use Sonarqube.

Reply:
Thanks @dlvenable for your review, Removed.